src/Store/CookieStore.php

```diff
@@ -13,29 +14,70 @@ class CookieStore implements StoreInterface
     const BASE_NAME = 'auth0_';
 
     /**
-     * Cookie base name, configurable on instantiation.
+     * Cookie base name.
+     * Use config key 'base_name' to set this during instantiation.
      *
      * @var string
      */
-    protected $cookie_base_name;
+    protected $baseName;
 
     /**
-     * Cookie expiration, configurable on instantiation.
+     * Cookie expiration length.
+     * This will be added to current time or $this->now to set cookie expiration time.
+     * Use config key 'expiration' to set this during instantiation.
      *
      * @var integer
      */
-    protected $cookie_expiration;
+    protected $expiration;
+
+    /**
+     * SameSite attribute for all cookies set with class instance.
+     * Must be one of None, Strict, Lax (default is no SameSite attribute).
+     * Use config key 'samesite' to set this during instantiation.
+     *
+     * @var null|string
+     */
+    protected $sameSite;
+
+    /**
+     * Time to use as "now" in expiration calculations.
+     * Used primarily for testing.
+     * Use config key 'now' to set this during instantiation.
+     *
+     * @var null|integer
+     */
+    protected $now;
+
+    /**
+     * Support legacy browsers for SameSite=None.
+     * This will set/get/delete a fallback cookie with no SameSite attribute if $this->sameSite is None.
+     * Use config key 'legacy_samesite_none' to set this during instantiation.
+     *
+     * @var boolean
+     */
+    protected $legacySameSiteNone;
 
     /**
      * CookieStore constructor.
      *
-     * @param string  $base_name Cookie base name.
-     * @param integer $expires   Cookie expiration length, in seconds.
+     * @param array $options Cookie options. See class properties above for keys and types allowed.
      */
-    public function __construct(string $base_name = self::BASE_NAME, int $expires = 600)
+    public function __construct(array $options = [])
     {
-        $this->cookie_base_name  = $base_name;
-        $this->cookie_expiration = $expires;
+        $this->baseName   = $options['base_name'] ?? self::BASE_NAME;
+        $this->expiration = $options['expiration'] ?? 600;
```

Comment:
Is this default documented anywhere?  Kind of expected to see it on line 27 above.